sklearn/model_selection/tests/test_split.py

```diff
@@ -616,7 +617,7 @@ def test_stratified_shuffle_split_overlap_train_test_bug():
     y = [0, 1, 2, 3] * 3 + [4, 5] * 5
     X = np.ones_like(y)
 
-    splits = StratifiedShuffleSplit(n_iter=1,
```

Comment:
`splits` --> `sss`

(This is a bit unrelated to the PR... But since you are touching the code, you might as well change it...)